### DISCUSSION TITLE
Add related content to blogs

### DISCUSSION THREAD

**Discussion by newbeelearn** [gethinode/hinode#812] [Ideas]
It would be great if related content can be added to blog based on tags/categories/keywords or some other criteria. Right now content is displayed based on dates.

**Comment by markdumay** [comment]
Just to clarify, are you looking for **related content** or **regular lists**? Right now, Hinode does not include related content, although it's not difficult to add. Regarding regular lists, I'm working on some modifications, but that's not ready to be released yet. Meanwhile, you could consider to use taxonomies instead of regular lists to get what you have in mind. See [release v0.21.0](https://github.com/gethinode/hinode/releases/tag/v0.21.0) for more details.

**Reply by newbeelearn** [reply]
i am looking for related content. It looks like a small thing but it is quite important. I recently discovered while checking my analytics that a significant percent of views were generated by related content.

**Reply by markdumay** [reply]
You might want to check the Hugo docs: https://gohugo.io/content-management/related/#list-related-content. I've added feature request https://github.com/gethinode/hinode/issues/815, so it would be a matter of configuration.

**Reply by newbeelearn** [reply]
great thanks for considering it.
